app/models/middleware_server_group.rb

```diff
@@ -1,6 +1,11 @@
 class MiddlewareServerGroup < ApplicationRecord
   belongs_to :middleware_domain, :foreign_key => "domain_id"
   has_many :middleware_servers, :foreign_key => "server_group_id", :dependent => :destroy
+  has_many :middleware_deployments, :foreign_key => "server_group_id", :dependent => :destroy
   serialize :properties
   acts_as_miq_taggable
+
+  def ext_management_system
+    try(:middleware_domain).try(:ext_management_system)
+  end
```

Comment:
@Jiri-Kremser could you `delegate :ext_management_system, to: :middleware_domain, :allow_nil => true`?